packages/e2e-tests/config/setup-test-framework.js

```diff
@@ -148,6 +147,40 @@ function observeConsoleLogging() {
 	} );
 }
 
+/**
+ * Runs Axe tests when the block editor is found on the current page.
+ *
+ * @return {?Promise} Promise resolving once Axe texts are finished.
+ */
+async function runAxeTestsForBlockEditor() {
+	if ( ! await page.$( '.block-editor' ) ) {
+		return;
+	}
+
+	await expect( page ).toPassAxeTests( {
+		// Temporary disabled rules to enable initial integration.
+		// See: https://github.com/WordPress/gutenberg/pull/15018.
```

Comment:
It would be good to have these in a tracking issue, perhaps structured as a checklist to iteratively mark progress.

Reply:
Yes, I will do it after merging this PR.

Reply:
#15452 opened